sql/database.sql

```diff
@@ -4744,6 +4752,27 @@ INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUE
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','Calendar','main/calendar/index.php',20,0,0);
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','oeSignerRemote','./../portal/sign/assets/signit.php',30,0,0);
 
+--Inserting data for sex,sexual orientation, gender identity categories, A5 items
+INSERT INTO `list_options`(`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('sex','HL7:UNK','Unknown',3,0,0);
+
+INSERT INTO `list_options`(`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('sexual_orientation','SNOMED:20430005','Straight or heterosexual',1,0,0);
```

Comment:
The option_id can not contain special characters. Recommend for the option_id to just be everything to right of the colon. And then populate the codes column with what you have now in the option_id.

Reply:
for the seq, start at 10 and then increment by 10.